Content.IntegrationTests/Tests/GameRules/NukeOpsTest.cs

```diff
@@ -50,6 +50,9 @@ public async Task TryStopNukeOpsFromConstantlyFailing()
         var invSys = server.System<InventorySystem>();
         var factionSys = server.System<NpcFactionSystem>();
 
+        // test urist is a noob let him be nukie
+        server.CfgMan.SetCVar(CCVars.GameRoleTimers, false);
+
```

Comment:
Shouldn't this just be disabled globally anyway.

Reply:
the cvar in code is true by default but on debug(?) its set to false somehow except in tests idk

also itd probably break a lot of servers

Reply:
also to fix test for real how to make urist opt into being commander

Reply:
Tests don't automatically reset cvars, so you have to manually reset the cvar back to its original value at the end of the test, or you need to change the value for all tests in `PoolManager.Cvars.cs`.

Reply:
cvar is now being reset, only issue is still the opt in part to fix tests